src/ducks/transfers/TransferPage.spec.jsx

```diff
@@ -133,7 +133,14 @@ describe('personal info', () => {
       const { root } = setup({
         myselfData: {
           birthcity: 'Compiègne',
-          nationality: 'FR'
+          nationality: 'FR',
+          cozyMetadata: {
+            updatedByApps: [
+              {
+                slug: 'slug'
```

Comment:
Does it mean that `manifest.slug` === "slug" when testing?

Reply:
Yep.